fakturogen-invoice/src/main/java/pl/fakturogen/invoice/dao/repository/InvoiceRepository.java

```diff
@@ -1,7 +1,4 @@
 package pl.fakturogen.invoice.dao.repository;
 
-import org.springframework.data.jpa.repository.JpaRepository;
-import pl.fakturogen.invoice.dao.entity.Invoice;
-
-public interface InvoiceRepository extends JpaRepository<Invoice, Long> {
+public class InvoiceRepository {
```

Comment:
we should have here previous version with extends JpaRepository, version with current change is step backwards